lib/src/test/java/io/ably/lib/test/realtime/RealtimePresenceTest.java

```diff
@@ -2062,7 +2067,8 @@ public void onPresenceMessage(PresenceMessage message) {
 				}
 			});
 
-			PresenceMessage[] testData = new PresenceMessage[] {
+			/* Test message newness criteria as described in RTP2b */
+			final PresenceMessage[] testData = new PresenceMessage[] {
```

Comment:
We could add a message that has an earlier timestamp, but a later msgSerial, expecting it to be added, so we confirm that that serial takes precedence over the timestamp.

We could also add message serials that explicitly test that the comparison is numeric instead of lexicographic, eg "9" < "10"